style.scss

```diff
@@ -765,6 +1034,7 @@ LATOUT
 
 //latest post section
 .latest-post-section {
+  position: relative;
   background-color: rgb(179, 212, 252);
   padding-top: 5rem;
   padding-left: 4rem;
```

Comment:
#### Use Shorthand Properties
Shorthand properties allow you to set the rules of numerous CSS properties simultaneously. Using a shorthand property will shrink your CSS and increase readability.

```
#div {

margin: 10px 10px 3px 4px; // top, right, bottom left 

}

Reply:
thx